css-isolation/package.json

```diff
@@ -1,5 +1,12 @@
 {
   "name": "css-isolation",
+  "private": true,
+  "workspaces": {
+    "packages": [
+      "./app1",
+      "./app2"
+    ]
+  },
```

Comment:
This will break the ci. 

When I wrote that deps don't install in sample subdir after run `yarn start` command, I mean run this in the root of the repo. 

But this changes will break CI, namely how it happens now you've changed the sample but lerna don't find it cause it private and e2e tests didn't start, as you can see

<img width="1661" alt="Screenshot 2023-01-24 at 09 01 45" src="https://user-images.githubusercontent.com/55971575/214231015-a3b2b0ab-93c3-4153-b105-9f08fb2b8c4b.png">